tests/roxctl/bats-tests/local/roxctl-netpol-connectivity-map-development.bats

```diff
@@ -636,6 +636,46 @@ monitoring/mymonitoring[Pod] => foo/myfoo[Pod] : All Connections'
   assert_output "$normalized_expected_output"
 }
 
+@test "roxctl-development netpol connectivity map generates explainability report" {
```

Comment:
add also a test that runs with `--explain` for unsupported format.

Reply:
done